packages/Telephony/Schemas/Microsoft.Telephony.FixedLengthBatchInput.uischema

```diff
@@ -0,0 +1,49 @@
+﻿{
+  "$schema": "https://schemas.botframework.com/schemas/ui/v1.0/ui.schema",
+  "form": {
+    "label": "Aggregate Input",
+    "subtitle": "Termination Character",
```

Comment:
We can use a more descriptive sub title here....

Reply:
Fixed